src/import/importConfiguration.ts

```diff
@@ -0,0 +1,713 @@
+    parameters.push({
+      code: 'copyrightyear',
+      value: `${yamlConfig.copyrightYear ?? yamlConfig.copyrightyear}`
+    });
+  }
+  if (yamlConfig.releaseLabel || yamlConfig.releaselabel) {
+    parameters.push({
+      code: 'releaselabel',
+      value: `${yamlConfig.releaseLabel ?? yamlConfig.releaselabel}`
+    });
+  }
+  if (yamlConfig.parameters) {
+    for (const [code, values] of Object.entries(yamlConfig.parameters)) {
+      normalizeToArray(values).forEach(value => parameters.push({ code, value: `${value}` }));
+    }
+  } else if (parameters.length === 0) {
+    return; // return undefined rather than an empty []
+  }
+  return parameters;
+}
+
+function parseTemplates(
+  yamlTemplates: ImplementationGuideDefinitionTemplate | ImplementationGuideDefinitionTemplate[],
+  file: string
+): ImplementationGuideDefinitionTemplate[] {
+  return normalizeToArray(yamlTemplates)?.map(t => {
+    const template: ImplementationGuideDefinitionTemplate = {
+      ...t,
+      code: parseSimpleCode(required(t.code, 'templates.code', file), 'templates.code', file),
+      source: required(t.source, 'templates.source', file)
+    };
+    removeUndefinedValues(template);
+    return template;
+  });
+}
+
+function parseMenu(yamlMenu: YAMLConfigurationMenuTree): ConfigurationMenuItem[] {
+  if (yamlMenu == null) {
+    return;
+  }
+  return Object.entries(yamlMenu).map(([name, value]) => {
+    const item: ConfigurationMenuItem = { name };
+    if (typeof value === 'string') {
+      item.url = value;
+    } else {
+      item.subMenu = parseMenu(value);
+    }
+    return item;
+  });
+}
+
+function parseHistory(yamlConfig: YAMLConfiguration, file: string): ConfigurationHistory {
+  const yamlHistory = yamlConfig['history'];
+  if (yamlHistory == null) {
+    return;
+  }
+  const history: ConfigurationHistory = {
+    'package-id': yamlHistory['package-id'] ?? yamlConfig.packageId ?? yamlConfig.id,
+    canonical: yamlHistory.canonical ?? yamlConfig.url,
+    title: yamlHistory.title ?? yamlConfig.title,
+    introduction: yamlHistory.introduction ?? yamlConfig.description,
+    list: []
+  };
+  if (yamlHistory.current) {
+    if (typeof yamlHistory.current === 'string') {
+      history.list.push({
+        version: 'current',
+        desc: 'Continuous Integration Build (latest in version control)',
+        path: yamlHistory.current,
+        status: 'ci-build',
+        current: true
+      });
+    } else {
+      history.list.push({
+        version: 'current',
+        date: normalizeToString(yamlHistory.current.date),
+        desc:
+          yamlHistory.current.desc ?? 'Continuous Integration Build (latest in version control)',
+        path: yamlHistory.current.path,
+        changes: yamlHistory.current.changes,
+        status: parseCodeWithRequiredValues(
+          yamlHistory.current.status ?? 'ci-build',
+          allowedHistoryStatus,
+          'history[current].status',
+          file
+        ),
```

Comment:
`path` should be.  I'll fix that.  Since we're in the context of `current`, if `status` isn't specified it defaults to `ci-build` so we don't need to mark it required.